extensions/wikia/CreateNewWiki/CreateNewWikiController.class.php

```diff
@@ -386,11 +404,14 @@ public function Phalanx() {
 
 		wfRunHooks( 'CheckContent', array( $text, &$blockedKeyword ) );
 
-		$this->msgHeader = '';
-		$this->msgBody = '';
 		if ( !empty( $blockedKeyword ) ) {
-			$this->msgHeader = wfMsg('cnw-badword-header');
-			$this->msgBody = wfMsg('cnw-badword-msg', $blockedKeyword);
+			$this->info( __METHOD__ . ": keyword blocked by Phalanx '" . $text . "'': " . $blockedKeyword );
```

Comment:
I am huge - and probably the only - fan of `sprintf`. In this case I would rely on interpolation and use concatenation only to add method.

`$this->info( __METHOD__ . ": keyword blocked by Phalanx '$text': $blockedKeyword"`

But it is readable as it is, kind of like English? It is easy to guess what it does :-)